### PR TITLE
add owner panel with facade

### PR DESCRIPTION
**Pull request by Zeicks**

### CODE REVIEW

File: src/main/java/com/easyrest/components/owner/Administrators.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package com.easyrest.components.owner;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class Administrators {private WebDriver driver;
```
Comment (martadiachok) [inline src/main/java/com/easyrest/components/owner/Administrators.java]: move please Webdriver to line below

File: src/main/java/com/easyrest/components/owner/CreateNewEmployee.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    private final By phoneField = By.xpath("//input[@name='phone_number']");
+    private final By addButton = By.xpath("//*[text() = 'Add']");
+    private final By cancelButton = By.xpath("//*[text() = 'Cancel']");
+
+    public CreateNewEmployee(WebDriver driver) {
+        this.driver = driver;
+    }
+
+    //We know that we have code duplication
+    public CreateNewEmployee sendToFieldName(String name) {
```
Comment (martadiachok) [inline src/main/java/com/easyrest/components/owner/CreateNewEmployee.java:23]: maybe named method "inputName"to make it more simple?

Comment (martadiachok) [reply src/main/java/com/easyrest/components/owner/CreateNewEmployee.java:23]: the same comment for 3 methods below

File: src/main/java/com/easyrest/components/owner/Details.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
+package com.easyrest.components.owner;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class Details {
```
Comment (martadiachok) [inline src/main/java/com/easyrest/components/owner/Details.java]: as for me, better name is "RestaurantDetails"

File: src/main/java/com/easyrest/pages/OwnerPanel.java [hunk 4]
```diff
@@ -1,311 +1,41 @@
...
-        return new OwnerPanel(driver);
-    }
-
-    public OwnerPanel clickOnButtonDeleteAdministrator() {
-
-        driver.findElement(deleteAdministratorButton).click();
-        return new OwnerPanel(driver);
-    }
-}
+        addRestaurant = new AddRestaurant(driver);
```
Comment (martadiachok) [inline src/main/java/com/easyrest/pages/OwnerPanel.java]: do you need all of this if you have facade?

File: src/main/java/com/easyrest/components/owner/Administrators.java (hunk 1 above)
Comment (Zeicks) [reply src/main/java/com/easyrest/components/owner/Administrators.java]: Done

File: src/main/java/com/easyrest/components/owner/CreateNewEmployee.java (hunk 2 above)
Comment (Zeicks) [reply src/main/java/com/easyrest/components/owner/CreateNewEmployee.java:23]: yes, inputName it more simple but sendToFieldName has more description what we do

File: src/main/java/com/easyrest/components/owner/Details.java (hunk 3 above)
Comment (Zeicks) [reply src/main/java/com/easyrest/components/owner/Details.java]: Done

File: src/main/java/com/easyrest/pages/OwnerPanel.java (hunk 4 above)
Comment (Zeicks) [reply src/main/java/com/easyrest/pages/OwnerPanel.java]: cleaned

**Review by martadiachok — APPROVED** [review]

File: src/main/java/com/easyrest/components/owner/Administrators.java [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+    private final By addAdministratorButton = By.xpath("//button[@title = 'Add Administrator']");
```
Comment (vadymtsehelny) [inline src/main/java/com/easyrest/components/owner/Administrators.java:8]: Code formatting

File: src/main/java/com/easyrest/components/owner/Administrators.java (hunk 1 above)
Comment (vadymtsehelny) [inline src/main/java/com/easyrest/components/owner/Administrators.java]: Code formatting

File: src/main/java/com/easyrest/components/owner/CreateNewEmployee.java [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+    }
+
+    public CreateNewEmployee sendToFieldPassword(String password) {
+        driver.findElement(passwordField).clear();
+        driver.findElement(passwordField).sendKeys(password);
+        return this;
+    }
+
+    public CreateNewEmployee sendToFieldPhone(String phone) {
+        driver.findElement(phoneField).clear();
```
Comment (vadymtsehelny) [inline src/main/java/com/easyrest/components/owner/CreateNewEmployee.java:42]: DRY

**Review by vadymtsehelny — APPROVED** [review]

File: src/main/java/com/easyrest/components/owner/CreateNewEmployee.java [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+        driver.findElement(phoneField).sendKeys(phone);
+        return this;
```
Comment (vadymtsehelny) [inline src/main/java/com/easyrest/components/owner/CreateNewEmployee.java:44]: DRY

Comment (vadymtsehelny) [inline src/main/java/com/easyrest/components/owner/CreateNewEmployee.java:44]: DRY

File: src/main/java/com/easyrest/components/owner/CreateNewEmployee.java [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+
+    public CreateNewEmployee sendToFieldPassword(String password) {
+        driver.findElement(passwordField).clear();
+        driver.findElement(passwordField).sendKeys(password);
+        return this;
+    }
+
+    public CreateNewEmployee sendToFieldPhone(String phone) {
+        driver.findElement(phoneField).clear();
+        driver.findElement(phoneField).sendKeys(phone);
```
Comment (vadymtsehelny) [inline src/main/java/com/easyrest/components/owner/CreateNewEmployee.java:43]: DRY

File: src/main/java/com/easyrest/components/owner/Administrators.java (hunk 1 above)
Comment (Zeicks) [reply src/main/java/com/easyrest/components/owner/Administrators.java]: it's already fixed

File: src/main/java/com/easyrest/components/owner/CreateNewEmployee.java (hunk 8 above)
Comment (Zeicks) [reply src/main/java/com/easyrest/components/owner/CreateNewEmployee.java:43]: I know this, we decided to do this on the daily meeting

**Review by ViktoriaMilkevich — APPROVED** [review]

File: src/main/java/com/easyrest/pages/OwnerPanel.java [hunk 9]
```diff
@@ -1,311 +1,34 @@
...
-    private final By coffeTeaRestTag = By.xpath("//li [@data-value = 'coffe/tea']");
-
-    private final By restaurantDescriptionField = By.xpath("//label [text()='Restaurant Description']");
-    private final By addButton = By.xpath("//span[text()='Add']");
-
-    private final By deleteWaiterButton = By.xpath("//li[contains (@class, 'MuiListItem-root')][1]//*[@viewBox='0 0 24 24']");
-    private final By administratorsMenuButton = By.xpath("//span[text() = 'Administrators']");
-    private final By addAdministratorButton = By.xpath("//button[@title = 'Add Administrator']");
-    private final By deleteAdministratorButton = By.xpath("//li[contains (@class, 'MuiListItem-root')][1]//*[@viewBox='0 0 24 24']");
+    private Right_sideMenu right_sideMenu;
```
Comment (dmatsev) [inline src/main/java/com/easyrest/pages/OwnerPanel.java]: What is the purpose of having private components in the page object?

**Review by dmatsev — COMMENTED** [review]
I left the comment.

**Review by dmatsev — APPROVED** [review]

**Pull request merged**